students/1st/feature:1st:hayounggam/js/login.js

```diff
@@ -0,0 +1,44 @@
+const idInput = document.querySelector("#js-id-input"),
+  pwInput = document.querySelector("#js-pw-input"),
+  btnLogin = document.querySelector(".btn-login");
+
+const GREEN_COLOR = "#00704a";
+const GREY_COLOR = "#dddddd";
+
+const toggleBtnLogin = () => {
+  const idLength = idInput.value.length;
+  const pwLength = pwInput.value.length;
+
+  idLength > 0 && pwLength > 0
+    ? (btnLogin.disabled = false)
+    : (btnLogin.disabled = true);
+};
+
+const changeIdInputBorder = () => {
+  if (idInput.value.includes("@")) {
+    idInput.style.borderColor = GREEN_COLOR;
+  } else {
+    idInput.style.borderColor = GREY_COLOR;
+  }
+};
+
+const changePwInputBorder = () => {
+  if (pwInput.value.length >= 8) {
+    pwInput.style.borderColor = GREEN_COLOR;
+  } else {
+    pwInput.style.borderColor = GREY_COLOR;
+  }
+}
```

Comment:
class 이름 추가/제거로 스타일을 변경하시는 로직으로 바꿀 때도 이런식으로 하면 좋을 것 같네요.
```suggestion
const changePwInputBorder = () => {
  pwInput.style.borderColor = pwInput.value.length >=8 ? GREEN_COLOR : GREY_COLOR;
}
```

Reply:
장현님 지적 감사드립니다🙇‍♀️
[input border-color 동적 스타일 변경](https://github.com/wecode-bootcamp-korea/webucks-javascript/pull/9#discussion_r644442662)의 지적사항과 함께 코드를 수정했습니다. 확인 부탁드립니다!

https://github.com/wecode-bootcamp-korea/webucks-javascript/pull/9/commits/0d7b3935ac1ed835c5fc01288882473b2491b521